setup.cfg

```diff
@@ -87,7 +80,7 @@ match_dir = ^(?!\.|doc|examples).*$
 add-ignore = D100,D104,D107,D413
 add-select = D214,D215,D404,D405,D406,D407,D408,D409,D410,D411
 ignore-decorators = property
-match = 
+match =
     (?!_eeg_features).*\.py
```

Comment:
@adam2392 This looks like an old match for one of the files that were in the `tests` folder. Also, if this is not on a separate line but on the same line -> `match = (?!_eeg_features).*\.py`, `pydocstyle` complains a lot about our current docstrings. 
I guess this match should be updated with an exclusion for the `tests` .py files?

Reply:
Ah good catch... then the pydocstyle was passing all along mistakenly? :(